src/Microsoft.Azure.WebJobs.Logging/Internal/Utility.cs

```diff
@@ -72,5 +74,80 @@ public static string DecodeBase64(string base64)
             string str = Encoding.UTF8.GetString(bytes);
             return str;
         }
+
+
+        private static async Task SafeCreateAsync(this CloudTable table)
```

Comment:
I have an idea to unit test the while(true) part. I'll include in next iteration.